kubernetes/resource_kubernetes_service_account.go

```diff
@@ -169,6 +169,10 @@ func resourceKubernetesServiceAccountRead(d *schema.ResourceData, meta interface
 	if err != nil {
 		return err
 	}
+	err = d.Set("automount_service_account_token", *svcAcc.AutomountServiceAccountToken)
```

Comment:
This crashes with a panic if svcAcc.AutomountServiceAccountToken is nil.
It should be checked for `nil` before using it, as with all pointer attributes.